.github/actions/pytest/action.yml

```diff
@@ -16,6 +16,7 @@ runs:
       shell: bash -l {0}
       run: |
         source ~/.bash_profile
```

Comment:
Maybe you add a comment line explaining the need for the following two lines? Mainly for posteriority, since this will be the only ODD action in an otherwise very regular and repetitive definition structure.

Reply:
was removed again :)